rct229/data_fns/table_3_2_fns.py

```diff
@@ -38,12 +39,16 @@ def table_3_2_lookup(climate_zone_enum_val):
     Returns
     -------
     dict
-        { system_min_heating_ouput: float – the heating output for the given climate one given by Table 3.2 [Btu/h-ft^2] }
+        { system_min_heating_ouput: Quantity – the heating output for the given climate zone given by Table 3.2 [Btu/h-ft^2] }
     """
     climate_zone = climate_zone_enumeration_to_climate_zone_map[climate_zone_enum_val]
     osstd_entry = find_osstd_table_entry(
         [("climate_zone", climate_zone)],
         osstd_table=data["ashrae_90_1_table_3_2"],
     )
-    system_min_heating_output = osstd_entry["Btu/h-ft^2"]
+    btu_per_hour_per_sqft = osstd_entry["Btu/h-ft^2"]
+    system_min_heating_output = btu_per_hour_per_sqft * ureg(
+        "british_thermal_unit / foot ** 2 / planck_constant"
```

Comment:
I don't think we need the `planck_constant`.

Should be "british_thermal_unit / (hour * foot ** 2)"